features/s3/postObject.feature

```diff
@@ -0,0 +1,9 @@
+@s3 @integ
+Feature: POST object uploads
+
+    Scenario: Upload and object via POST
+        Given I have an "s3" client
+        And I have a file
+        When I create a POST object
```

Comment:
This needs to be a bit more explicit. Cucumber features are both tests and documentation for how to interact with the code. You might want to change this to either a series of steps or something like "I create a POST object with the following conditions:" followed by a a string that you can JSON-decode.